### PR TITLE
Changed to Grid layout, Breakpoints tweaked

### PR DESCRIPTION
**Pull request by uqtgeor4**

### CODE REVIEW

File: packages/footer/src/scss/_component.scss [hunk 1]
```diff
@@ -59,46 +59,31 @@
...
-            }
-        }
-
-        &:last-child {
-            padding: core.$space-none;
-        }
-
+        display: grid;
+        grid: auto-flow / 1fr;
+        gap: 1.5rem;
```
Comment (uqnm) [inline packages/footer/src/scss/_component.scss:64]: this could be increased a little more timmy for medium if possible?

Comment (uqtgeor4) [reply packages/footer/src/scss/_component.scss]: Done. Gap increased between columns

Comment (uqnm) [reply packages/footer/src/scss/_component.scss]: sorry timmy - few more minor things here... we may as well adjust these now while we are looking at them. Could we tweak the vertical spacing/paddings for the elements when on mobile? More so to evenly space the elements. I can talk you through this if you need more info.

Comment (uqtgeor4) [reply packages/footer/src/scss/_component.scss]: Done

**Review by uqnm — APPROVED** [review]
will setup a new ticket for mobile vertical spacing

**Pull request merged**